### DISCUSSION TITLE
queue prompts

### DISCUSSION THREAD

**Discussion by beepbeepplop** [deforum-art/sd-webui-text2video#157] [Ideas]
I love this extension, but queuing is so important to me. Please can we get prompt queuing (I want to queue 100 of each prompt over night). Thanks

**Comment by kabachuha** [comment]
I know 😣

There's already the framework, need to find the time to add it

**Reply by beepbeepplop** [reply]
great. and can we add some info to the names of the files so they're not all the same. i'm generating 100 at a time and i don't want to overwrite them. thanks. awesome work.

**Reply by beepbeepplop** [reply]
Please can we have the ability to queue different prompts. Modelscope is still unrivalled in free ai video generation and I would be using it a lot more if I didn't have to sit and prompt every video. Please?
